spec/controllers/admin_posts_controller_spec.rb

```diff
@@ -44,9 +44,21 @@
         end
       end
 
-      context "when admin has correct authorization" do
-        %w[superadmin board communications support translation].each do |admin_role|
-          context "with #{admin_role} role" do
+      context "with invalid translated_post_id" do
```

Comment:
Updating a post would fail without correct admin roles, and the params of the update make no difference, so I don't think this test is necessary. I think we can have just one test under "when admin does not have correct authorization", and remove both nested contexts "with valid title" and "with invalid translated_post_id".

Reply:
Can do! I was just shuffling what already existed so I didn't have to add a third "when admin has correct authorization" section, but I'm game for deleting existing tests.